src/components/table/table.js

```diff
@@ -1177,27 +1181,28 @@ export default {
           // We need to add a hint about what the column is about for non-dighted users
           ariaLabel = startCase(field.key)
         }
-        const ariaLabelSorting = field.sortable
+        const sortable = field.sortable && (!isFoot || !this.noFooterSorting)
```

Comment:
This could be simply written as:

```js
const sortable = field.sortable && !(isFoot && this.noFooterSorting)
```

Which is a bit more clear with it's intention